content-src/components/SectionMenu/SectionMenu.jsx

```diff
@@ -12,6 +12,10 @@ export class _SectionMenu extends React.PureComponent {
     const {props} = this;
 
     const propOptions = props.isWebExtension ? [...WEBEXT_SECTION_MENU_OPTIONS] : [...DEFAULT_SECTION_MENU_OPTIONS];
```

Comment:
For discovery stream we should hide 'Remove Section' link menu option from ...DEFAULT_SECTION_MENU_OPTIONS

Reply:
Confirmed that we want to keep all the menu options including remove for 68.